tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     /* Basic Options */
-    "target": "ES5" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */,
+    "target": "es2017" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */,
```

Comment:
Why did this need to change?

Reply:
bec if doesn't work to do `err instanceof IoTsValidationError` if target `ES5`.
err was always instance of `Error`

Reply:
Do you need to go all the way to es2017 it es2015 is enough?
You don’t want users of the lib to need to transpile.

Reply:
Yes, that works too.. I'll update.